app/core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object"""
+    name = models.CharField(max_length=31)
```

Comment:
31? Any specific reason? :sweat_smile:  (I'd not asked for 42)

Reply:
I wanted something small to force titles of recipes to be short, the description field can be used for full context. In the training course the fields are all 255 which is a bit too long, perhaps I've been a bit too mean with the default value set here.

^^ incidentally, I just checked and that comment is 254 chars long 🤣